src/graphnet/models/components/embedding.py

```diff
@@ -70,7 +71,8 @@ def __init__(
         Args:
             seq_length: Dimensionality of the base sinusoidal positional
                 embeddings.
-            output_dim: Output dimensionality of the final projection.
+            mlp_dim: Hidden dimensionality of the final Mlp.
```

Comment:
"mlp_dim (Optional): Size of hidden, latent space of MLP. If not given, `mlp_dim` defaults to `n_features`."